spec/configuration.yml.example

```diff
@@ -7,11 +7,11 @@ root:
 user:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
```

Comment:
These config are only for specs. We only connect as `root`.

In fact, the `user` config is unused and, according to git history, has never been used!

IMO the `numericuser` config is inappropriate here too. The spec should use the same config and just change the db name to a numeric one.